translations/stripes-erm-components/en_US.json

```diff
@@ -197,7 +197,11 @@
     "errorLogNo": "No errors to report.",
     "columns.errorLogMessage": "Error",
     "columns.infoLogMessage": "Message",
-    "columns.recordNumber": "Record number",
+    "columns.recordDescriptor": "Record descriptor",
+    "packageSource": "Package source: {source}",
+    "packageReference": "Package reference: {ref}",
+    "rowNumber": "Row number: {rownumber}",
+    "recordNumber": "Record number: {recordnumber}",
```

Comment:
Only translations that are already present have to be edited in the `en_US.json` file. New translations `shouldn't` be added here and adding them only to `en.json` should suffice.